### PR TITLE
Docs: landing page for OSS/Enterprise

### PR DESCRIPTION
**Pull request by brendamuir**
Adds new landing page for OSS and Enterprise docs

### CODE REVIEW

**Comment by brendamuir** [comment]
@jdbaldry this page has a banner note about Grafana Cloud...i am not sure whether i deleted it or left it - could you check? https://grafana.com/docs/grafana/latest/. i think we still want it near the top

**Comment by jdbaldry** [comment]
I don't see any banner on the page https://grafana.com/docs/grafana/latest/. Might it be coming from marketing rather than the docs source?

File: docs/sources/alerting/fundamentals/alert-rules/annotation-label.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Examples of labels are `server=server1` or `team=backend`. Each alert rule can h
 
 For example, an alert instance might have the label set `{alertname="High CPU usage",server="server1"}` while another alert instance might have the label set `{alertname="High CPU usage",server="server2"}`. These are two separate alert instances because although their `alertname` labels are the same, their `server` labels are different.
 
-{{< figure src="/static/img/docs/alerting/unified/multi-dimensional-alert.png" >}}
+{{< figure alt="Alert instance labels" src="/static/img/docs/alerting/unified/multi-dimensional-alert.png" >}}
```
Comment (jdbaldry) [inline docs/sources/alerting/fundamentals/alert-rules/annotation-label.md:71]: This alt text should be more descriptive and explain what is in the image as if you were describing it to someone over the phone: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#alt-text

File: docs/sources/_index.md [hunk 2]
```diff
@@ -6,137 +6,84 @@ aliases:
 cascade:
   TEMPO_VERSION: latest
   PYROSCOPE_VERSION: latest
-description: Guides, installation, and feature documentation
+description: Landing page for Grafana OSS and Enterprise documentation
```
Comment (jdbaldry) [inline docs/sources/_index.md:9]: I'm not sure if this description works as a call to action for users to click on this page's link if it comes up in social media or search engines. 

Including the product name is a good improvement though.

**Review by jdbaldry — APPROVED** [review]
Looks much better, thanks!

**Pull request merged**